.github/workflows/cd.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish logstash-oss-with-opensearch-plugin
```

Comment:
Nit: Could cd.yml be renamed to CD.yml? It seems that is the convention for plugins.

Reply:
Ack